### PR TITLE
Update 0082-helium-mobile-service-provider.md

### PR DESCRIPTION
**Pull request by zer0tweets**
Updating HIP with the following: 

- Grace period changed to 6 months from one year 
- If opt-out not implemented past 6 months, reward cap goes away until implementation is complete 
- Reward cap is adjusted every year based to be 1 standard deviation away from average subscriber (per KeithR proposal)

### CODE REVIEW

File: 0082-helium-mobile-service-provider.md
Comment (abhay) [inline 0082-helium-mobile-service-provider.md:73]:
```suggestion
Additionally, we propose a 6 calendar month grace period to allow for unrewarded traffic over 30GB per Subscriber on the Hotspots. Nova Labs will implement an option to opt-out of unrewarded data traffic in a Cloud Dashboard or similar tool no later than the expiration of the grace period. 6 month countdown will start on the day this HIP is approved.  
```

**Review by abhay — CHANGES_REQUESTED** [review]

**Comment by vincenzospaghetti** [comment]
@abhay - your change was accepted! Need your final review before we're able to merge.

**Review by abhay — APPROVED** [review]

**Comment by abhay** [comment]
> @abhay - your change was accepted! Need your final review before we're able to merge.

typo recommendation. merging.

**Pull request merged**
